package.json

```diff
@@ -130,6 +130,12 @@
       }
     },
     "rules": {
+      "@typescript-eslint/no-unused-vars": [
```

Comment:
Curious why it's a `@typescript-eslint` rule, isn't this a normal js rule anyway?

Reply:
It's preferred for compat with types. It's generally recommended to [disable the base rule](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unused-vars.md) but I assume this is already handled by one of our presets. Only setting `no-unused-vars` will invoke both.

Reply:
Thanks for the explanation.